pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
+filterwarnings =
+    # Fail the tests if there are any warnings.
+    error
+
```

Comment:
In the case where the tests fail in the normal way, this line often obscures the real reason. For example a perfectly normal test failure will often now look like it was caused a warning encountered on the way. 

I'm not sure if the information is actually gone, or just harder to find. I've certainly been toggling this on and off when trying to work out what really stopped the tests.